editor/static/sharing/index.js

```diff
@@ -0,0 +1,12 @@
+function share(program){
+    if (program =="whatsapp"){
+        var whatsapp = "https://web.whatsapp.com/send?text=```";
+        var text = savedTabInfo[focusedTab].text;
```

Comment:
now this will only work if tabs are used, and this will use the saved text, not the current text.

If your idea was to save the current text then you can just do `var text = getEditorText()` and this will work in all situations.

If you actually want to use the saved text then you can just set the text to savedText, this works with the latest commit on the tabs branch.

Reply:
This can now be changed, as the tab fixes were merged! @huynguy97